tests/data/examples_create_labware_messages.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+from typing import Dict, Any
+
+TEST_CREATE_LABWARE_MSG_OBJECT: Dict[str, Any] = {
+    "messageUuid": "b01aa0ad-7b19-4f94-87e9-70d74fb8783c".encode(),
+    "messageCreateDateUtc": datetime.now(),
```

Comment:
If the host machine is not at UTC, this call is locale dependent. Maybe it should be datetime.utcnow()

Reply:
Additionally, in general, I can't understand how the date-time serialisation works in these messages.

Reply:
Yes I think it should be `.utcnow()`.  The serialisation is handled automatically by the `fastavro` library based on the Avro message schema that was defined.

Reply:
This is the date when the message was generated at source (by TOL team). This Dict is a testing message, that would be the result of unserialising the message from the queue which it will generate a datetime object in this Dict